openmdao/core/vec_wrapper_complex_step.py

```diff
@@ -0,0 +1,298 @@
+""" VecWrapper 'wrapper' that is used for component-wise complex step."""
+
```

Comment:
That would definitely result in less code.  I have a feeling that this code will go away later anyway when it's time to implement a general complex step solution that will work under MPI.